src/zjs_sensor.c

```diff
@@ -245,6 +245,9 @@ static void zjs_sensor_update_reading(jerry_value_t obj,
         return;
     }
 
+    uint32_t timestamp = k_uptime_get();
```

Comment:
k_uptime_get returns a uint64_t - if you only need the bottom part use k_uptime_get_32() - it's faster at least.

Otherwise LGTM.